app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -29,7 +29,12 @@ snippet: true
 
   # tell Anaconda what to pass off to kickstart server
   # both current and legacy syntax provided
-  options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
+
+  if (is_fedora && os_major >= 34) || (rhel_compatible && os_major >= 9)
```

Comment:
Please remove the extra line, the comment above does belong to lines 33-37 actually.

Reply:
The extra line should now be removed.